pyccel/ast/core.py

```diff
@@ -819,23 +819,31 @@ def int2float(expr):
 def float2int(expr):
     return expr
 
-def create_random_string(expr):
+def create_random_string(forbidden_exprs, prefix = 'Dummy', counter = 1):
```

Comment:
The output string is not "random" anymore, correct? 😉

Reply:
I have renamed it `create_incremented_string`